### DISCUSSION TITLE
Capacitor App does not work.

### DISCUSSION THREAD

**Discussion by selfee-jp** [quasarframework/quasar#9372] [CLI - Capacitor / Cordova modes] [answered]
Quasar is a very good frame worm.
Unfortunately, it is still not widely used in Japan.
I would like to support it.

"Capacitor App" is not working.
It's an empty new project. 1.

1. quasar create test
2. quasar mode add capacitor

It is working in Android studio.
It doesn't work in the debug build apk.
The screen is still blank. Normally, the Quasar logo should be displayed.

**Comment by rstoenescu** [comment, marked as answer]
I can assure you that it works, regardless of Quasar version (v1 with Vue 2 or v2 with Vue 3).
It's all in the details. Can we see the full list of commands that you are using? Also, which version of Quasar are you using?

**Reply by rstoenescu** [reply]
Just throwing my two cents at what happened for you. The "quasar dev -m capacitor -T [..]" fills the /src-capacitor/www folder with a setup that only works as long as the "quasar dev" command is still running. Once you kill it, you'll indeed get the blank screen because the app installed on the emulator (or real phone) does not have the devserver to connect to anymore.

**Reply by selfee-jp** [reply]
The version of Quasar is v1.

"quasar dev -m capacitor -T [...]" After execution
 "quasar dev" was terminated.
The problem seems to have been solved. Let's give it a try.

Are there any conditions when releasing with "quasar build -m capacitor -T [ios|android]"?


Actually, I was planning to use Quasar with "MONACA".
https://monaca.io/
I was able to install Quasar on "MONACA".
However, the same problem occurred on "MONACA".

I switched to building with "Quasar CLI".
Since I don't have IOS, "MONACA" came in handy.

I hope this forum will be of help to me.
Thank you very much.
[package.zip](https://github.com/quasarframework/quasar/files/6526603/package.zip)

**Reply by selfee-jp** [reply]
It worked with "quasar dev".
This seems to be the web server displaying the www folder. Is there any way to test this as a complete build application?
I need it to test push notifications. Or can I test it as is?

**Reply by ontwikkelfabriek** [reply]
@bscg0283 
> I need it to test push notifications. Or can I test it as is?

You'll need to run you Quasar Capacitor app in a non dev mode, preferably on a `real` device to be able to test if `push notifications` work for a Capacitor app on Android and iOS.

> Since I don't have IOS, "MONACA" came in handy.

I use Vmware to run a VM with MacOS on a Windows host to build/test IOS apps.( and to publish to the iTunes store) 

>It doesn't work in the debug build apk.

Did you read this:
_If developing on a mobile phone/tablet, it is very important that the external IP address of your build machine is accessible from the phone/tablet, otherwise you’ll get a development app with white screen only. Also check your machine’s firewall to allow connections to the development chosen port._ 

Its' from https://quasar.dev/quasar-cli/developing-capacitor-apps/build-commands

**Reply by selfee-jp** [reply]
Thanks, @ontwikkelfabriek ontwikkelfabriek .
I was able to run the "apk" in debug mode.

I like the idea of Vmware.
I'm going to adopt that idea.

By the way.
I heard that push notifications need to be run in non-development mode.
Do I need to sign the app for testing purposes?

It seems I can't install it on my device without signing it.

**Reply by metalsadman** [reply]
@bscg0283 you can install the bundle on a device, using debug flag `quasar build -m capacitor -T android --debug`. test it otherwise, try a signed apk.

**Reply by selfee-jp** [reply]
Thank you, @metalsadman .
I will try it soon.

**Reply by kamalsprasad** [reply]
> @metalsadman thank you! that worked for me with the debug flag. When I tried with signed apk (without the debug flag) I got a "You can't install this app on your device" error
